--- login.html	2021-05-20 10:32:56.000000000 -0600
+++ login.html.FIXED	2021-09-27 08:35:06.000000000 -0600
@@ -54,33 +54,12 @@
         Token authentication is enabled
       </h3>
       <p>
-        If no password has been configured, you need to open the notebook
-        server with its login token in the URL, or paste it above.
-        This requirement will be lifted if you
-        <b><a href='https://jupyter-notebook.readthedocs.io/en/stable/public_server.html'>
-            enable a password</a></b>.
-      </p>
-      <p>
-        The command:
-        <pre>jupyter notebook list</pre>
-        will show you the URLs of running servers with their tokens,
-        which you can copy and paste into your browser. For example:
-      </p>
-      <pre>Currently running servers:
-http://localhost:8888/?token=c8de56fa... :: /Users/you/notebooks
-</pre>
-      <p>
-        or you can paste just the token value into the password field on this
-        page.
-      </p>
-      <p>
-        See
-        <b><a
-         href='https://jupyter-notebook.readthedocs.io/en/stable/public_server.html'>
-                the documentation on how to enable a password</a>
-        </b>
-        in place of token authentication,
-        if you would like to avoid dealing with random tokens.
+        You need to open the notebook server with the token provided by the notebook.
+        This can be found in the log for this container.
+        <pre>
+          kubectl logs deployment/jupyter -n <i>INSTANCE-NAME</i>
+        </pre>
+        Copy and past the token value in the password field above.
       </p>
       <p>
         Cookies are required for authenticated access to notebooks.
